packages/tag/index.ts

```diff
@@ -1 +1 @@
-import './Tag';
+import './TagSelector';
```

Comment:
both need to be imported:

```
import './Tag';
import './TagSelector';
```

 in next PR whole structure should be refactored to common pattern

See https://github.com/wc-catalogue/blaze-elements/issues/191